lib/api/bucketGetTagging.js

```diff
@@ -0,0 +1,90 @@
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { pushMetric } = require('../utapi/utilities');
+const { errors } = require('arsenal');
+
+//  Sample XML response:
+/*
+    <Tagging>
+        <TagSet>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
```

Comment:
Maybe it can be worth adding more than one tag in the example tagset ? What do you think ?

Reply:
done https://github.com/scality/cloudserver/pull/4481/commits/bc8c6f527ec83cfcc7cb7b2bcf2d83ff9b35609b#diff-889d6969db5c5fda428f1b3f508e237cb4bc1419949c78bc0a146bcca7d9209bR14